### PR TITLE
ocm: app-ui add skip if only changed

### PR DESCRIPTION
**Pull request by ray-harris**

### CODE REVIEW

File: ci-operator/jobs/open-cluster-management/application-ui/open-cluster-management-application-ui-main-presubmits.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-open-cluster-management-application-ui-main-e2e-cluster
     rerun_command: /test e2e-cluster
+    skip_if_only_changed: ^docs/|\.(md|adoc)$|^(README|LICENSE)$
```
Comment (KevinFCormier) [inline ci-operator/jobs/open-cluster-management/application-ui/open-cluster-management-application-ui-main-presubmits.yaml:17]: Did we want to include OWNERS here?

Comment (ray-harris) [reply ci-operator/jobs/open-cluster-management/application-ui/open-cluster-management-application-ui-main-presubmits.yaml:17]: Probably. Let me get it passing all the tests first.

**Pull request merged**